status_app.py

```diff
@@ -213,6 +213,8 @@ def __init__(self, settings):
             ("/api/v1/workset_notes/([^/]*)$", WorksetNotesDataHandler),
             ("/api/v1/workset_links/([^/]*)$", WorksetLinksHandler),
             ("/workset/([^/]*)$", WorksetHandler),
+            ("/assign_roles", AssignRolesHandler),
```

Comment:
I think we aim to keep the urls in alphabetic order in this list.

Reply:
alles in Ordnung!